app/models/worker_job_application_record.rb

```diff
@@ -0,0 +1,7 @@
+# rubocop:disable Rails/ApplicationRecord
+class WorkerJobApplicationRecord < ActiveRecord::Base
```

Comment:
do we need coverage here?

Reply:
this class is actually used, the GoodJob::Job dynamically subclasses it, however that may be set up before the code coverage is set up

If this class was not used, GoodJob would look in the wrong DB for its table